test/integration/tcp_tunneling_integration_test.cc

```diff
@@ -183,6 +207,10 @@ TEST_P(ConnectTerminationIntegrationTest, TestTimeout) {
 }
 
 TEST_P(ConnectTerminationIntegrationTest, BuggyHeaders) {
+  if (downstream_protocol_ == Http::CodecClient::Type::HTTP1) {
+    // These buggy headers using :method require HTTP/2 or later.
```

Comment:
:method will be translated to normal HTTP/1.1 firstline so I'm not sure why this doesn't work for HTTP/1.1

Reply:
I don't know either, I removed the comment